### PR TITLE
Extend IsEmpty to SpanID, remove usage of IsValid

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

**Review by carlosalberto — APPROVED** [review]

File: consumer/pdata/spanid.go [hunk 1]
```diff
@@ -40,6 +40,12 @@ func (t SpanID) HexString() string {
...
 
 // IsValid returns true if id contains at leas one non-zero byte.
+// Deprecated: use !IsEmpty() instead.
 func (t SpanID) IsValid() bool {
 	return data.SpanID(t).IsValid()
 }
+
+// IsEmpty returns true if id doesn't contain at least one non-zero byte.
+func (t SpanID) IsEmpty() bool {
+	return !data.SpanID(t).IsValid()
```
Comment (tigrannajaryan) [inline consumer/pdata/spanid.go:50]: Should we also change `data.SpanID` to have `IsEmpty` instead of `IsValid`?

**Review by tigrannajaryan — APPROVED** [review]
I like the change. It is more aligned with Collector's vision, which says we are not in the business of deciding what is valid data and what is not (unless it is necessary to do our job).

File: consumer/pdata/spanid.go (hunk 1 above)
Comment (bogdandrutu) [reply consumer/pdata/spanid.go:50]: will do it in a followup PR.

**Pull request merged**